README.md

```diff
@@ -2,6 +2,11 @@
 
 Tool to generate load for [Temporal](https://github.com/temporalio/temporal/) workflows. Named after [Kobayashi Maru](https://en.m.wikipedia.org/wiki/Kobayashi_Maru).
 
+**⚠️ WARNING**
+
+This repository is a load generator for Temporal and does not represent suggested Go SDK practices or worker settings.
+Please reference the Go SDK documentation or the Go SDK samples for better guidance.
```

Comment:
Link?

Reply:
The former changes enough (just did recently from https://docs.temporal.io/docs/go -> https://docs.temporal.io/go but that's deprecated and they want you to link to https://docs.temporal.io/application-development-guide now) and this repo already suffers from staleness that I was afraid to do so. And linking the more stable samples without the other seems inconsistent.